modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -672,22 +572,7 @@ public CompletableFuture<Void> startAsync(ComponentContext componentContext) {
 
             cleanUpResourcesForDroppedTablesOnRecoveryBusy();
 
-            if (!nodeProperties.colocationEnabled()) {
-                metaStorageMgr.registerPrefixWatch(
-                        new ByteArray(PENDING_ASSIGNMENTS_QUEUE_PREFIX_BYTES),
-                        pendingAssignmentsRebalanceListener
-                );
-                metaStorageMgr.registerPrefixWatch(new ByteArray(STABLE_ASSIGNMENTS_PREFIX_BYTES), stableAssignmentsRebalanceListener);
-                metaStorageMgr.registerPrefixWatch(
-                        new ByteArray(ASSIGNMENTS_SWITCH_REDUCE_PREFIX_BYTES),
-                        assignmentsSwitchRebalanceListener
-                );
-            }
-
-            catalogService.listen(
-                    CatalogEvent.TABLE_CREATE,
-                    nodeProperties.colocationEnabled() ? onTableCreateWithColocationListener : onTableCreateWithoutColocationListener
-            );
+            catalogService.listen(CatalogEvent.TABLE_CREATE, onTableCreateWithColocationListener);
```

Comment:
I'll rename it but a bit later in order not produce unnecessary mess in given PR.